xml/html_title.xml

```diff
@@ -1695,10 +1710,11 @@
   </fingerprint>
 
   <fingerprint pattern="^Login - OpenStack Dashboard$">
-    <description>OpenStack Dashboard</description>
+    <description>OpenStack Horizon Dashboard</description>
     <example>Login - OpenStack Dashboard</example>
     <param pos="0" name="service.vendor" value="OpenStack"/>
-    <param pos="0" name="service.product" value="Dashboard"/>
+    <param pos="0" name="service.product" value="Horizon"/>
```

Comment:
Incorrect product name but calling it out here in case anyone depends on it.

Reply:
Nexpose side does not depend on these